mantidproject/mantid
# Indirect - IqtFit - Display ties added by 'Constrain Intensities' in the Fit Property Browser

Currently, in the Indirect IqtFit interface, the tie created by the 'Constrain Intensities' option does not appear in the browser (although it is used in the fit).

This PR ensures the fit property browser displays the tie.

**Must be merged after PR #21962**

**To test:**
1. Navigate to Interfaces > Indirect > Data Analysis > IqtFit.
2. Add two exponential functions using the 'Exponential' spinner and a stretched exponential using the 'Stretched Exponential' checkbox.
3. Select 'Flat Background' from the 'Background Type' drop-down menu.
4. Check the check-box labelled 'Constrain Intensities'.
5. Ensure the tie can be viewed in the function browser found under the 'Functions' heading. The tie should be each of the Height parameters of the exponential functions and the A0 parameter of the background function, all subtracted from 1.
6. Uncheck the check-box labelled 'Constrain Intensities'.
7. Ensure the tie is removed.



Fixes #21807. 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
